Chargebee/Classes/Restore/CBRestoreResource.swift

```diff
@@ -0,0 +1,80 @@
+
+//
+//  CBRestoreServiceManager.swift
+//  Chargebee
+//
+//  Created by ramesh_g on 20/02/23.
+//
+
+import Foundation
+import StoreKit
+
+final class CBRestoreResource: CBAPIResource {
```

Comment:
- Can rename it to `CBRestorePurchaseResource`
- Please update the comments in all files.

Reply:
i have rename the file done.